tests/test_settings.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from roadmap.config import Settings
+
+
+async def test_settings(monkeypatch):
+    assert os.environ.get("ACG_CONFIG") is None
+    assert os.environ.get("ROADMAP_DB_USER") is None
+
+    from roadmap.config import SETTINGS
+
+    assert SETTINGS.db_user == "postgres"
+
+    monkeypatch.setenv("ROADMAP_DB_USER", "test_db_user")
+    assert Settings.create().db_user == "test_db_user"
+
+    monkeypatch.setenv("ACG_CONFIG", os.path.join(os.getcwd(), "tests", "fixtures", "clowder_config.json"))
```

Comment:
This is brittle because it will change depending on where the test is executed from. A better strategy is to use a relative path from `__file__`.
```suggestion
    monkeypatch.setenv("ACG_CONFIG", Path(__file__).parent.joinpath("fixtures").resolve().join("clowder_config.json"))
```

Reply:
I'm curious why resolve is in the middle there. Also I had to change 'join' to 'joinpath', but this is updated now.